app/lib/date-helpers.js

```diff
@@ -225,6 +233,12 @@ const removeIndividualDateComponents = (payload) => {
   return payload
 }
 
+const getEndOfDayTime = (inDate) => {
+  const endOfDay = new Date(inDate)
+  endOfDay.setTime(endOfDay.getTime() + (24 * 60 * 60 * 1000) - 1000)
+  return endOfDay
+}
+
```

Comment:
this will be helpful as well going forward.  We should probably copy this to api.